src/ServiceControl/ServiceControl.csproj

```diff
@@ -656,7 +655,12 @@
       <Install>false</Install>
     </BootstrapperPackage>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <ProjectReference Include="..\ServiceControl.LearningTransport\ServiceControl.LearningTransport.csproj">
+      <Project>{12f4b317-b885-4d43-a152-fdb5b0ea2042}</Project>
+      <Name>ServiceControl.LearningTransport</Name>
+    </ProjectReference>
+  </ItemGroup>
```

Comment:
What's this?

Reply:
That's not supposed to be in there. I will update the PR when I get home.

Reply:
;)